### PR TITLE
ADEN-5056 Fix Veles tracking

### PR DESCRIPTION
**Pull request by fraszczakszymon**
- always define bidResponse, no matter if it's valid or not
- refactor "used" bids
- mark slots which are not supposed to take Veles response

### CODE REVIEW

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/veles.js [hunk 1]
```diff
@@ -155,11 +160,21 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.veles', [
...
+			}
+		});
+	}
+
 	return {
 		create: create,
 		isEnabled: isEnabled,
 		getName: getName,
 		getSlots: getSlots,
+		markBidsAsUsed: markBidsAsUsed,
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/veles.js:177]: Can we unit test it?

File: extensions/wikia/AdEngine/js/lookup/prebid/adaptersPerformanceTracker.js [hunk 2]
```diff
@@ -78,16 +78,26 @@ define('ext.wikia.adEngine.lookup.prebid.adaptersPerformanceTracker', [
 	}
 
 	function getParamsFromBidForTracking(bid) {
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/lookup/prebid/adaptersPerformanceTracker.js:80]: Can we unit test it? :japanese_ogre:

Comment (nandy-andy) [reply extensions/wikia/AdEngine/js/lookup/prebid/adaptersPerformanceTracker.js:80]: Oh, it's not public :/ We'll test it while testing integration then, I guess ([ADEN-5062](https://wikia-inc.atlassian.net/browse/ADEN-5062)).

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/veles.js (hunk 1 above)
Comment (nandy-andy) [reply extensions/wikia/AdEngine/js/lookup/prebid/adapters/veles.js:177]: Done: https://github.com/Wikia/app/pull/12809/commits/70bdbc3c341da66bd684dfa6ebaf16e39af0d9ea

**Review by dianafa — APPROVED** [review]

**Review by nandy-andy — APPROVED** [review]

**Pull request merged**